src/components/Thanks/ThanksPageShare.vue

```diff
@@ -198,6 +220,10 @@ export default {
 			type: String,
 			default: 'a'
 		},
+		categoryShareVersion: {
+			type: String,
+			default: 'c'
```

Comment:
Is this default no experiment?

Reply:
Yes, we're handling two variants c is default!